src/ducks/apps/components/MyApplications.jsx

```diff
@@ -68,10 +70,13 @@ const MyApplicationsWrapper = props => {
   const isExact = useMatch('myapps')
   const navigate = useNavigateNoUpdates()
   const [searchParams] = useSearchParams()
+  const { alternativeApps } = useAlternativeStore()
+  const installedAltApps = alternativeApps.filter(app => app.installed)
```

Comment:
Maybe we could add this filter into the useAlternativeStore so we would be able to reuse it in other place